Modules/Scripted/DICOMLib/DICOMUtils.py

```diff
@@ -668,3 +668,35 @@ def onNodeAdded(caller, event, calldata):
   slicer.mrmlScene.RemoveObserver(sceneObserverTag)
 
   return loadedNodeIDs
+
+def loadFromDICOMWeb(dicomWebEndpoint, studyUID, seriesUID="", accessToken=""):
```

Comment:
Unless there is a strong reason to do otherwise, use None as default parameter.

Reply:
I think the correct letter case for the method should be `loadFromDICOMweb` for consistency with the [official trademark style](https://www.dicomstandard.org/dicomweb/).  Then the variable name would be `dicomWEB`.  (I know, it looks awkward - perhaps we make a style exception here.)

Reply:
I don't think that trademark style would apply to code (only to displayed text) but if we change the method name to "loadFromDICOMweb" then add a comment there that this is intentional (otherwise someone may "fix" it when comes across the coding style violation).

DICOMWeb and DICOMweb are about equally bad (hard to read/inconsistent with other) so I don't have a strong preference there. But dicomWeb looks more friendly and consistent to me than dicomWEB (it almost shouts at me).

Reply:
Agreed!  I've started using 'dicom' more often than DICOM in regular text to avoid that extra screaming element.  

Anyway yeah, let's leave it as is.